openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -4693,10 +4689,10 @@ def create_force(self, system, topology, **kwargs):
         # To keep it simple, we DO NOT pre-populate the particles in the GBSA force here.
         # We call addParticle further below instead.
         # These lines are commented out intentionally as an example of what NOT to do.
-        # for topology_particle in topology.topology_particles:
+        # for particle in topology.particles:
         # gbsa_force.addParticle([0.0, 1.0, 0.0])
 
-        params_to_add = [[] for _ in topology.topology_particles]
+        params_to_add = [[] * topology.n_particles]
```

Comment:
Interesting - I broke this because of a gotcha I'm un familiar with - multiply an empty list doesn't work in the same as as multiplying one with contents:

```
ipdb> [[] for x in range(2)]
[[], []]
ipdb> [[] * 2]
[[]]
ipdb> [['ok'] for x in range(2)]
[['ok'], ['ok']]
ipdb> [['ok'] * 2]
[['ok', 'ok']]